app/src/lib/emails/confirmRegistrationTemplate.tsx

```diff
@@ -0,0 +1,150 @@
+import { useCopyToClipboard } from "@/hooks/use-copy-to-clipboard";
+import { CityAttributes } from "@/models/City";
+import { UserAttributes } from "@/models/User";
+import React, { useState } from "react";
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Html,
+  Link,
+  Preview,
+  Text,
+  Img,
+  Font,
+  Hr,
+} from "@react-email/components";
+
+export default function ConfirmRegistrationTemplate({
+  url = "/",
+  user = { name: "" },
+}: {
+  url?: string;
+  user?: { name: string };
+}) {
+  const ImageURL = "https://citycatalyst.openearth.dev/assets/icon.png";
+  return (
+    <Html>
+      <Head>
+        <Font
+          fontFamily="Roboto"
+          fallbackFontFamily="Verdana"
+          webFont={{
+            url: "https://fonts.gstatic.com/s/roboto/v27/KFOmCnqEu92Fr1Mu4mxKKTU1Kg.woff2",
+            format: "woff2",
+          }}
+          fontWeight={400}
+          fontStyle="normal"
+        />
+      </Head>
+
+      <Preview>CityCatalyst: User Registration</Preview>
+      <Body style={main}>
+        <Container style={container}>
+          <div style={content}>
+            <Img
+              src={ImageURL}
+              alt="City Catalyst logo"
+              width="36"
+              height="36"
+            />
+            <Text style={brandHeading}>CityCatalyst</Text>
+            <Text style={heading}>Welcome to CityCatalyst</Text>
+            <Text style={greeting}>Hi {user?.name},</Text>
+            <Text style={paragraph}>
+              Thank you for registering an account CityCatalyst. <br /> <br />{" "}
+              Please activate your account by clicking this button below. <br />{" "}
+              <br />
+            </Text>
+
+            <div
+              style={{
+                marginTop: "36px",
+                marginBottom: "36px",
+              }}
+            >
+              <Link href={url} style={urlLink}>
+                ACTIVATE ACCOUNT
+              </Link>
+            </div>
+
+            <Hr style={{ height: "2px", background: "#EBEBEC" }} />
+            <Text style={footerText}>
+              Open Earth Foundation is a nonprofit public benefit corporation
+              from California, USA. EIN: 85-3261449
+            </Text>
```

Comment:
Can we start using i18n here as well? @isaaccodekill set up our i18n system to be available on the server as well AFAIK.